tests/unit/test_pathutil.py

```diff
@@ -574,3 +577,43 @@ def test_get_workflow_name_from_id(
 
     result = get_workflow_name_from_id(id_)
     assert result == name
+
+
+@pytest.fixture
+def _setup_get_source_conf_from_id(tmp_path, monkeypatch):
+    run = tmp_path / 'cylc-run/run'
+    src = tmp_path / 'cylc-src/src'
+    src.mkdir(parents=True)
+    (run / '_cylc-install').mkdir(parents=True)
+    (run / '_cylc-install/source').symlink_to(src)
+    monkeypatch.setattr(
+        'cylc.flow.pathutil.get_workflow_run_dir',
+        lambda workflow_id: tmp_path / 'cylc-run/run'
+    )
+    monkeypatch.setattr(
+        'cylc.flow.pathutil.get_cylc_run_dir',
+        lambda: tmp_path / 'cylc-run'
+    )
+    yield tmp_path
```

Comment:
Any reason not to just use the `tmp_run_dir` and `tmp_src_dir` fixtures?

Reply:
All a moot point since I deleted the function this is testing.